c7n/manager.py

```diff
@@ -111,8 +111,10 @@ def filter_resources(self, resources, event=None):
         original = len(resources)
         if event and event.get('debug', False):
             self.log.info(
-                "Filtering resources with %s", self.filters)
+                "Filtering resources using %d filters", len(self.filters))
+        c = 0
         for f in self.filters:
+            c = c + 1
```

Comment:
`for idx, f in enumerate(self.filters)` seems simpler, and perhaps more pythonic

Reply:
ah yeah... i knew there had to be a better way...